LBH.AdultSocialCare.Api/V1/Infrastructure/Entities/Common/CarePackage.cs

```diff
@@ -1,54 +1,35 @@
+using LBH.AdultSocialCare.Api.Attributes;
+using LBH.AdultSocialCare.Api.V1.AppConstants.Enums;
+using LBH.AdultSocialCare.Api.V1.Domain.Common;
+using LBH.AdultSocialCare.Api.V1.Infrastructure.Entities.NursingCare;
+using LBH.AdultSocialCare.Api.V1.Infrastructure.Entities.ResidentialCare;
 using System;
 using System.Collections.Generic;
 using System.ComponentModel.DataAnnotations;
 using System.ComponentModel.DataAnnotations.Schema;
-using LBH.AdultSocialCare.Api.V1.AppConstants.Enums;
-using LBH.AdultSocialCare.Api.V1.Infrastructure.Entities.NursingCare;
-using LBH.AdultSocialCare.Api.V1.Infrastructure.Entities.ResidentialCare;
-
-using PackageStatusNew = LBH.AdultSocialCare.Api.V1.AppConstants.Enums.PackageStatus;
 
 namespace LBH.AdultSocialCare.Api.V1.Infrastructure.Entities.Common
 {
+    [GenerateMappingFor(typeof(CarePackagePlainDomain))]
+    [GenerateListMappingFor(typeof(CarePackagePlainDomain))]
     public class CarePackage : BaseEntity
     {
-        [Key, DatabaseGenerated(DatabaseGeneratedOption.Identity)]
```

Comment:
I believe that having spaces between properties increase readability, especially in classes with lot of attributes, since all mouses have scroll wheels, and without spaces it hard to see where attributes end and properties start, but up to you..

Will continue adding spaces anyway.. )

Reply:
I do find it easy to work with it this way but yeah, any approach should be fine I guess